WordPress/src/main/java/org/wordpress/android/ui/people/PeopleAdapter.java

```diff
@@ -77,8 +75,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
             holder.imgAvatar.setImageUrl(avatarUrl, WPNetworkImageView.ImageType.AVATAR);
             holder.txtDisplayName.setText(person.getDisplayName());
             holder.txtUsername.setText(String.format("@%s", person.getUsername()));
-            holder.txtRole.setText(Role.getLabel(mContext, person.getRole()));
-            holder.txtRole.setBackgroundColor(Role.backgroundColor(mContext, person.getRole()));
+            holder.txtRole.setText(StringUtils.capitalize(person.getRole()));
```

Comment:
Capitalize changes the first letter, `textAllCaps` changes all. So, we can't use that ;)